spk/zerotier/src/dsm-control.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/sh
+
+DNAME="ZeroTier"
+PACKAGE="zerotier"
+INSTALL_DIR="/usr/local/${PACKAGE}"
+ZT_HOME_DIR="/var/lib/zerotier-one"
+ZEROTIER="${INSTALL_DIR}/bin/zerotier-one"
+WATCHDOG="${INSTALL_DIR}/bin/zerotier-watchdog.sh"
+PID_FILE="${ZT_HOME_DIR}/zerotier-one.pid"
+LOG_FILE="/var/log/zerotier-one.log"
+
+start_daemon ()
+{
+    ${WATCHDOG} stop ;
```

Comment:
I do not understand interest of watchdog process in addition to service itself. What does it run for?